<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LimitedUnlocks</class>
 <widget class="QWidget" name="LimitedUnlocks">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string comment="The title of the 'Limited Unlocks' tab in the editor.">Limited Unlocks</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="chkAssassinHoodUnlock">
     <property name="text">
      <string comment="Whether the Assassin's Hood is unlocked.">Assassin's Hood Unlocked</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkAssassinHoodUpgrade">
     <property name="text">
      <string comment="Whether the Assassin's Hood is upgraded.">Assassin's Hood Upgraded</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkLayeredArtemis">
     <property name="text">
      <string comment="Whether the Artemis Layered is unlocked.">Layered: Artemis (MR required)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkLayeredBayek">
     <property name="text">
      <string comment="Whether the Bayek Layered is unlocked.">Layered: Bayek</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnGiveArtemisGear">
     <property name="text">
      <string comment="Add Artemis Gear in the player's equipment box.">Give Artemis Gear</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnGiveYukumoLoadout">
     <property name="text">
      <string comment="Add a preset layered loadout for Yukumo.">Add Yukumo Layered Loadout</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnGiveSilverKnightLoadout">
     <property name="text">
      <string comment="Add a preset layered loadout for Silver Knight.">Add Silver Knight Layered Loadout</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnGiveSamuraiLoadout">
     <property name="text">
      <string comment="Add a preset layered loadout for Samurai.">Add Samurai Layered Loadout</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>chkAssassinHoodUpgrade</sender>
   <signal>stateChanged(int)</signal>
   <receiver>LimitedUnlocks</receiver>
   <slot>UpgradeAssassinHood(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkAssassinHoodUnlock</sender>
   <signal>stateChanged(int)</signal>
   <receiver>LimitedUnlocks</receiver>
   <slot>UnlockAssassinHood(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnGiveArtemisGear</sender>
   <signal>clicked()</signal>
   <receiver>LimitedUnlocks</receiver>
   <slot>GiveArtemisGear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkLayeredArtemis</sender>
   <signal>stateChanged(int)</signal>
   <receiver>LimitedUnlocks</receiver>
   <slot>UnlockLayeredArtemis(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkLayeredBayek</sender>
   <signal>stateChanged(int)</signal>
   <receiver>LimitedUnlocks</receiver>
   <slot>UnlockLayeredBayek(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnGiveYukumoLoadout</sender>
   <signal>clicked()</signal>
   <receiver>LimitedUnlocks</receiver>
   <slot>GiveYukumoLoadout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnGiveSilverKnightLoadout</sender>
   <signal>clicked()</signal>
   <receiver>LimitedUnlocks</receiver>
   <slot>GiveSilverKnightLoadout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnGiveSamuraiLoadout</sender>
   <signal>clicked()</signal>
   <receiver>LimitedUnlocks</receiver>
   <slot>GiveSamuraiLoadout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>UnlockAssassinHood(int)</slot>
  <slot>UpgradeAssassinHood(int)</slot>
  <slot>GiveArtemisGear()</slot>
  <slot>UnlockLayeredArtemis(int)</slot>
  <slot>UnlockLayeredBayek(int)</slot>
  <slot>GiveYukumoLoadout()</slot>
  <slot>GiveSilverKnightLoadout()</slot>
  <slot>GiveSamuraiLoadout()</slot>
 </slots>
</ui>
